src/components/Conversion/ConversionSummary.vue

```diff
@@ -0,0 +1,171 @@
+<template>
```

Comment:
At the moment, this is almost identical to ChangeSummary.vue, but there are some differences between change and conversion filings that will be implemented here in the future.